### DISCUSSION TITLE
在broker load时，查询性能急剧下降，导致所有的查询超时！

### DISCUSSION THREAD

**Discussion by zyw8136** [apache/doris#7611] [A - General / Q&A]
在做broker load时，所有的查询性能急剧下降，这块应该如何优化，请大佬们给提供点排查问题的思路或是建议。
![image](https://user-images.githubusercontent.com/4756695/148167669-b1ddce2b-5ce0-4030-902f-9002e70a12c3.png)

**Comment by morningman** [comment]
Please paste a profile as an attachment here.

**Reply by zyw8136** [reply]
![broker load](https://user-images.githubusercontent.com/4756695/148312097-4ddb815a-d507-4191-8a97-ab2a0dec887e.png)
![query-profile-1](https://user-images.githubusercontent.com/4756695/148312136-e6149b13-e44e-445f-8589-626b459eba55.png)
![query-profile-2](https://user-images.githubusercontent.com/4756695/148312153-dac75886-00c4-4389-86f7-43f67108503d.png)

**Comment by zyw8136** [comment]
![screencapture-10-223-0-169-8030-QueryProfile-5c279e2dd0cf40e7-869a07b81518fecc-2022-01-06-11_54_54](https://user-images.githubusercontent.com/4756695/148325921-a56d2173-c81b-48d2-abd6-2f08a1ae9678.png)
![screencapture-10-223-0-169-8030-QueryProfile-5c279e2dd0cf40e7-869a07b81518fecc-2022-01-06-11_54_54-2](https://user-images.githubusercontent.com/4756695/148325960-ede97246-c934-4aa6-800e-a642cc1ac582.png)

这个查询比较有代表性！
